_config/config.yml

```diff
@@ -6,4 +6,7 @@ After:
 ---
 Controller:
   extensions:
-    - QuickFeedbackExtension
\ No newline at end of file
+    - QuickFeedbackExtension
+
+QuickFeedbackExtension:
```

Comment:
Docs are a good suggestion, and part of a separate pull request I'm working on...